packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -62,7 +63,7 @@ storiesOf('Caption', module)
         { name: 'caption', defaultText: 'Students sitting an examination' },
         { name: 'offscreen text', defaultText: 'Image Caption, ' },
       ],
-      ([captionText, offscreenText], script, dir) => (
+      ({ inputs: [captionText, offscreenText], script, dir }) => (
```

Comment:
Should this example have `service`?

Reply:
Yeah but we don't have service prop in the `Caption` component yet.